examples/nextjs-redux-whiteboard/pages/index.tsx

```diff
@@ -0,0 +1,148 @@
+import { useEffect, useMemo } from "react";
+import { useRouter } from "next/router";
+import { useDispatch, useSelector } from "react-redux";
+import { actions } from "@liveblocks/redux";
+import {
+  insertRectangle,
+  onShapePointerDown,
+  deleteShape,
+  onCanvasPointerUp,
+  onCanvasPointerMove,
+  client,
+  Shape,
+  State,
+  User
+} from "../src/store";
+import styles from "./app.module.css";
+
+let roomId = "nextjs-redux-whiteboard";
+
+export default function MyApp() {
+  useOverrideRoomId("roomId");
```

Comment:
```suggestion
let roomId = "redux-whiteboard";
useOverrideRoomId();

export default function MyApp() {
```
First change to overrideRoomId